ambry-store/src/main/java/com.github.ambry.store/BlobStoreCompactor.java

```diff
@@ -190,34 +198,46 @@ A single compaction job could be performed across multiple compaction cycles (if
     4. CLEANUP - Cleans up the old log segments and their index files
      */
 
+    logger.trace("Resuming compaction");
```

Comment:
"resumeCompaction() started" might be more helpful as it matches the end log message of this method.